frontend/javascripts/components/pricing_enforcers.tsx

```diff
@@ -82,7 +85,13 @@ export function PricingEnforcedButton({
 }
 
 export function PageUnavailableForYourPlanView() {
-  const organizationName = useSelector((state: OxalisState) => state.activeOrganization?.name);
+  const activeUser = useSelector((state: OxalisState) => state.activeUser);
+  const activeOrganization = useSelector((state: OxalisState) => state.activeOrganization);
+
+  const LinkToOrganizationSettings =
```

Comment:
usually, only component classes/functions are capitalized and not instantiations.

```suggestion
  const linkToOrganizationSettings =
```